pkg/ciContext/ciContext.go

```diff
@@ -73,6 +89,16 @@ func isCI() string {
 	return "none"
 }
 
+func HideEmojis() bool {
```

Comment:
I suggest renaming to ShouldHideEmojis for better readability, HideEmojis sounds as if what it does is actually hiding the emojis